Samples/WinMLSamplesGallery/WinMLSamplesGallery/Pages/SampleBasePage.xaml.cs

```diff
@@ -54,6 +57,11 @@ protected override async void OnNavigatingFrom(NavigatingCancelEventArgs e)
             {
                 var page = (Samples.Batching)SampleFrame.Content;
                 page.StopAllEvents();
+            } else if(SampleFrame.SourcePageType == typeof(Samples.DXResourceBindingORT))
+            {
+                var page = (Samples.DXResourceBindingORT)SampleFrame.Content;
+                page.StopAllEvents();
+
             }
```

Comment:
[nit] extra line. Propose moving this line break to line 60 instead :b.

}
else if (...)
{

Reply:
Done.